src/lang/english.txt

```diff
@@ -1234,6 +1234,8 @@ STR_CONFIG_SETTING_HORIZONTAL_POS_LEFT                          :Left
 STR_CONFIG_SETTING_HORIZONTAL_POS_CENTER                        :Centre
 STR_CONFIG_SETTING_HORIZONTAL_POS_RIGHT                         :Right
 
+STR_CONFIG_SETTING_SECONDS_VALUE                                :{COMMA}{NBSP}second{P 0 "" s}
```

Comment:
```suggestion
STR_CONFIG_SETTING_SECONDS_VALUE                                :{COMMA}{NBSP}second{P 0:2 "" s}
```

Reply:
The `:x` syntax for strings is (almost) only used with `{STRINGn}`, since those can have multiple parameters. With `{COMMA}` there is only one parameter.
That it was needed before was because the `{STRING2}` included a string containing `{COMMA}`, and from the string with the `{STRING2}` it wanted to know the plural for the value within the substring. In this case we are already in the substring, so we do not need to look within some sub-parameter.

Reply:
Ah, thanks. Is this new format the same as `{P "" s}` or is that different?

Reply:
`{P "" s}` and `{P 0 "" s}` are equivalent here.

Reply:
Thanks.